ext/js/data/options-util.js

```diff
@@ -1233,6 +1234,16 @@ export class OptionsUtil {
         }
     }
 
+    /**
+     *  - Added anki.duplicateBehavior
+     *  @type {import('options-util').UpdateFunction}
+     */
+    _updateVersion31(options) {
+        for (const {options: profileOptions} of options.profiles) {
+            profileOptions.anki.duplicateBehavior = 'prevent';
```

Comment:
It might be better to set the update setting to `new` so we arent "ping-ponging" around users' settings too much. Even if this setting will be `prevent` by default.